src/cloud-element-templates/linting/index.js

```diff
@@ -27,3 +30,18 @@ export const ElementTemplateLinterPlugin = function(templates) {
     })
   };
 };
+
+export const ElementTemplateCachedLinterPlugin = function(templates) {
+  return {
+    config: {
+      rules: {
+        'element-templates/validate-cached': [ 'error', { templates } ],
+        'element-templates/compatibility-cached': [ 'warn', { templates } ]
+      }
+    },
+    resolver: new StaticResolver({
+      'rule:bpmnlint-plugin-element-templates/validate-cached': validateCached,
+      'rule:bpmnlint-plugin-element-templates/compatibility-cached': compatibilityCached
+    })
+  };
+};
```

Comment:
I don't think we want to ship two versions of the plugin. If we decide to introduce caching, let's make it available for all.